README.md

```diff
@@ -25,7 +25,7 @@ the hoods to run benchmarks on JVM.
 
 Gradle 6.8 or newer
```

Comment:
Does the minimal supported Gradle version have Kotlin 1.5 bundled? If not, it may be not safe to use Kotlin stdlib 1.5 API in the code of the Gradle plugin.